contracts/Marketplace.sol

```diff
@@ -212,10 +215,6 @@ contract Marketplace is Collateral, Proofs, StateRetrieval {
     require(token.transfer(msg.sender, amount), "Withdraw failed");
   }
 
-  function getHost(SlotId slotId) public view returns (address) {
```

Comment:
We do need this for other purposes (checking whether a host filled a slot or another host got there first)